### PR TITLE
Added the python code for longest prefix suffix problem: Issue #499

### PR DESCRIPTION
**Pull request by Sanjsarma**

### CODE REVIEW

**Comment by rish-singhal** [comment]
provide issue number

**Comment by Sanjsarma** [comment]
Issue number is 499.

File: Code/Python/longest_prefix_suffix.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+'''
+
+#Longest prefix suffix function
+def longest_prefix_suffix(s):
+    length=len(s)
+    #Iterate i through half the length of string s
+    for i in range(length//2,0,-1): 
+        #Prefix is from 0th index to ith index
+        prefix=s[0:i]
```
Comment (rish-singhal) [inline Code/Python/longest_prefix_suffix.py:13]: give spaces around operators such as =, +, ....

File: Code/Python/longest_prefix_suffix.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    #Iterate i through half the length of string s
+    for i in range(length//2,0,-1): 
+        #Prefix is from 0th index to ith index
+        prefix=s[0:i]   
+        #Suffix is the remaining part of string. That is, length minus prefix
+        suffix=s[length-i:length] 
+        #If prefix and suffix are equal, return length of string. Else, return 0
+        if(prefix==suffix): 
+            return i;
+    return 0;
```
Comment (rish-singhal) [inline Code/Python/longest_prefix_suffix.py:19]: extra line needed

File: Code/Python/longest_prefix_suffix.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ 
+Input : rrrrr
+ 
+Output  : 2 
+
+Input : hellohe
+ 
+Output  : 2
+ 
+Time Complexity: O(n)
```
Comment (rish-singhal) [inline Code/Python/longest_prefix_suffix.py:42]: what is n? provide it in the code itself

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Comment by rish-singhal** [comment]
Also update read list linking your code

File: Code/Python/longest_prefix_suffix.py (hunk 3 above)
Comment (Sanjsarma) [reply Code/Python/longest_prefix_suffix.py:42]: n depends on the size of the input string. So it will be calculated only when user inputs the string.

**Pull request closed without merging**